src/main/resources/config/application-gcp.yml

```diff
@@ -0,0 +1,50 @@
+# ===================================================================
+# Spring Boot configuration for the "gcp" profile.
+#
+# This configuration overrides the application.yml file.
+#
+# More information on profiles: https://www.jhipster.tech/profiles/
+# More information on configuration properties: https://www.jhipster.tech/common-application-properties/
+# ===================================================================
+
+# ===================================================================
+# Standard Spring Boot properties.
+# Full reference is available at:
+# http://docs.spring.io/spring-boot/docs/current/reference/html/common-application-properties.html
+# ===================================================================
+
+spring:
+  cloud:
+    gcp:
+      sql:
+        enabled: true
+        database-name:
+        instance-connection-name:
+  datasource:
+    hikari:
+      data-source-properties:
+        socketFactory: com.google.cloud.sql.mysql.SocketFactory
+        cloudSqlInstance:
+      jdbcUrl:
+      username:
+      password:
+server:
+  port: ${PORT}
+jhipster:
+  security:
+    authentication:
+      jwt:
+        base64-secret:
+  mail: # specific JHipster mail property, for standard properties see MailProperties
+    enabled: true
+    from: jhipster-online@jhipster.tech
+    base-url: https://start.jhipster.tech
+application:
+  github:
+    client-id:
+    client-secret:
+    jhipster-bot-oauth-token:
+  gitlab:
+    client-id:
+    client-secret:
+    redirect-uri:
```

Comment:
unless these override a value provided in base application.yml or other profile-specific yml (prod/dev), I will prefer to not duplicate here.

Reply:
Thanks :+1: